telegram/poll.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, text, voter_count, **kwargs):
         self.text = text
         self.voter_count = voter_count
 
+        self._id_attrs = (self.text,)
```

Comment:
`voter_count` is required. should be added, as polls might have duplicate options.